ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryAdd.java

```diff
@@ -51,14 +51,16 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final String jndiName = ManagedExecutorServiceResourceDefinition.JNDI_NAME_AD.resolveModelAttribute(context, model).asString();
         final int priority = ManagedThreadFactoryResourceDefinition.PRIORITY_AD.resolveModelAttribute(context, model).asInt();
 
+        final ServiceName serviceName = ManagedThreadFactoryResourceDefinition.CAPABILITY.getCapabilityServiceName(name);
         final ManagedThreadFactoryService service = new ManagedThreadFactoryService(name, jndiName, priority);
-        final ServiceBuilder<ManagedThreadFactoryImpl> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedThreadFactoryServiceName(name), service);
+        final ServiceBuilder<ManagedThreadFactoryImpl> serviceBuilder = context.getServiceTarget().addService(serviceName, service);
```

Comment:
Use context.getCapabilityServiceTarget().addCapability(...) if possible.

Reply:
seems only possible for not dynamic caps

Reply:
done